service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -378,6 +378,13 @@ func (t *transferQueueActiveTaskExecutor) processCloseExecution(
 		},
 		children,
 	)
+
+	if err != nil {
+		// This is some retryable error, not NotFound or NamespaceNotFound.
+		return err
+	}
+
+	return nil
```

Comment:
isn't this the same as return err?

Reply:
It is. I am preparing for the next step here. There will be another function call in upcoming PR.